### ISSUE TITLE
Extension can not be loaded in ssh vscode server 1.54.3

### ISSUE THREAD

**Issue by trir262** [microsoft/vscode-jupyter#5342]
## Environment data

-   VS Code version: 1.54.3
-   Jupyter Extension version (available under the Extensions sidebar): v2021.3.684299474
-   Python Extension version (available under the Extensions sidebar): n/a
-   OS (Windows | Mac | Linux distro) and version: Oracle Linux Server 7.8
-   Python and/or Anaconda version: N/A
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): N/A
-   Jupyter server running: Local | Remote | N/A

## Expected behaviour

Extension can be installed offline

## Actual behaviour

Extension can not be installed offline

Error message in Log (Window)

```text
[2021-03-27 21:24:27.823] [renderer3] [error] Unable to install extension 'ms-toolsai.jupyter' as it is not compatible with VS Code '1.54.3'.: Error: Unable to install extension 'ms-toolsai.jupyter' as it is not compatible with VS Code '1.54.3'.
    at /home/`username`/.vscode-server/bin/2b9aebd5354a3629c3aba0a5f5df49f43d6689f8/out/vs/server/remoteExtensionHostAgent.js:108:97447
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

## Steps to reproduce:

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

1. install vscode 1.54.3 on client (Windows in my case)
2. install Remote extensions
3. Download Jupyter vsix and scp it to the linux server
4. Remote connect to linux server (will install vscode server on linux)
5. Go to extensions
6. Click the three dots on top and select `Install from vsix...`

## Logs

<details>

<summary>Output for <code>Jupyter</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Jupyter</code>)
</summary>

<p>

```
Not present since extension can not be installed
```

</p>
</details>

**Comment by joyceerhl** [comment]
@trir262 where are you downloading the Jupyter extension VSIX from? It's likely you've downloaded one that requires a more recent build of VS Code to be compatible with VS Code extension API changes. A workaround would be to install a more recent build of VS Code.

**Comment by oliverweber2** [comment]
I have the same issue, also with Remote WSL as well as Remote Docker Container. Both versions are the newest versions available for now.

**Comment by dx-coding** [comment]
Same issue here on RHEL 7.4/7.5/7.9.  VSCode 1.54.3 (latest at the time of this post).  My current work around is to use a previous version (2020.12.414227025)

**Comment by dx-coding** [comment]
There could be some issues with this plugin since VS Code 1.54.3 doesn't support the EL7 flavors anymore. See https://code.visualstudio.com/updates/v1_53#_electron-11-update

**Comment by joyceerhl** [comment]
Sorry, I just noticed that this issue pertains to [VS Code Server](https://github.com/cdr/code-server), not VS Code. The Jupyter extension [doesn't support](https://github.com/microsoft/vscode-jupyter/issues/5175) VS Code Server at the moment. I would suggest opening an issue on the code-server repository instead.

**Comment by oxy** [comment]
Hi there! I'm a dev on [cdr/code-server](https://github.com/cdr/code-server) - @trir262 (and the others in this issue) appear to be using the Microsoft remote extension / the official, proprietary `vscode-server` product from Microsoft, not our open-source downstream, `code-server`. (The similar product names can get confusing)

We currently don't support connecting to a `code-server` instance from a local install of vscode, and only expose vscode to the browser. #5175 was a downstream issue that I think we fixed by now, but we don't have anything to do with Microsoft's remote extensions / their proprietary remote server :sweat_smile:

**Comment by joyceerhl** [comment]
Thanks @oxy, I misread the original bug report. 

@trir262 would you mind letting me know where you downloaded the VSIX that you transfer to the remote via SCP? Additionally, could you please confirm whether this behavior persists with the latest 1.55 release of VS Code and v2021.5.702919634 of the Jupyter extension?

**Comment by trir262** [comment]
@joyceerhl I downloaded the ms-toolsai.jupyter-2021.5.689556049 VSIX file  from Microsoft VSCode extension marketplace: https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter.

I now tried with vscode insiders 1.56.0-insiders, opened an ssh connection to my linux and tried to install from VSIX the ms-toolsai.jupyter-2021.5.702919634 VSIX file and it worked like a charm!

Thanks for the quick resolution!
